client/apps/game/src/dojo/debounced-queries.ts

```diff
@@ -1,7 +1,13 @@
 import { HexPosition, ID } from "@bibliothecadao/types";
 import { Component, Metadata, Schema } from "@dojoengine/recs";
 import { ToriiClient } from "@dojoengine/torii-client";
-import { getBuildingsFromTorii, getEntitiesFromTorii, getOwnedArmiesFromTorii } from "./queries";
+import {
+  getBuildingsFromTorii,
+  getEntitiesFromTorii,
+  getOwnedArmiesFromTorii,
+  getQuestTilesFromToriiQuery,
```

Comment:
have same naming as others

Reply:
This is challenging as I am already calling `getTilesFromTorii` directly from `world.tsx` and need a name that separates the query from the debounced subscription adder.

I could bundle in the query into `debouncedGetQuestTilesFromTorii` as that is the only function calling it and will remove that name?